tests/test_db.py

```diff
@@ -412,19 +412,13 @@ def test_is_old_schema(tmp_path):
     responses.add(responses.GET, url_version, status=200, body="1")
     responses.add(responses.GET, url_version, status=200, body="42")
 
-    # When the remote version file doesn't exist, we consider the db as being old.
-    assert db.is_old_schema(db_path)
+    # When the remote version file doesn't exist, we consider the db as being new.
+    assert db.is_different_schema(db_path)
 
-    # When the remote version file doesn't exist, we consider the db as being old.
-    assert db.is_old_schema(db_path)
-
-    # When the remote version file exists and returns the same version as the current db, we consider the remote db as not being old.
-    assert not db.is_old_schema(db_path)
-
-    # When the remote version file exists and returns a newer version than the current db, we consider the remote db as not being old.
-    assert not db.is_old_schema(db_path)
```

Comment:
You should keep the same assertions as before here, nothing should have changed in these results

Reply:
Both lines are same ..

Reply:
They are the same, you are executing multiple times the same code but with different mocked responses (see the `responses.get` above).

The first response is an error with status code 404, so the DB doesn't exist.
The second response is an error with status code 424, so the DB doesn't exist (we want to make sure not only 404 results in not existing).
The third response is successful with result "1", so the DB exists and has version 1.
The fourth response is successful with result "42", so the DB exists and has version 42.